portfolio/src/main/webapp/community.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Community Page</title>
+    <script src="community.js"></script>
+    <link rel="stylesheet" href="community.css">
+  </head>
+  <body onload=renderComments()>
+    <div id="community-content">
+        <div id="comment-form-section">
+            <h2>Here are all the wonderful things people have to say about this website</h2>
+            <form action="/comments" method="POST">
+                <p>Username:</p>
+                <input name="username" />
+
+                <p>Comment</p>
+                <input name="comment" />
+                <br />
+
+                <input type="submit" />
+            </form>
+        </div>
+        <div id="submitted-comments">
```

Comment:
Indentation.

Reply:
Added comment so that div start was more clear